app/src/utils/ipfs.ts

```diff
@@ -0,0 +1,38 @@
+import { create } from 'ipfs-http-client';
+import config from 'src/config';
+import { GrantMetadata } from '@dgrants/types';
+
+const ipfs = create({
+  url: config.ipfs.endpoint,
+  headers: config.ipfs.headers,
+});
+
+export const createGrant = async ({ name, description }: GrantMetadata) => {
+  const res = await ipfs.add(JSON.stringify({ name, description }));
+  return res.cid;
+};
+
+export const createMetaPtr = ({ cid }: { cid: string }) => {
+  return `${config.ipfs.metaPtrEndpoint}/${cid}`;
+};
```

Comment:
Nit: maybe rename to `getMetaPtr` as this feels like a getter method that's not actually creating something new